app/_locales/en/messages.json

```diff
@@ -397,6 +397,12 @@
     "message": "Transak supports debit card and bank transfers (depending on location) in 59+ countries. $1 deposits into your MetaMask account.",
     "description": "$1 represents the cypto symbol to be purchased"
   },
+  "buyEth": {
+    "message": "Buy ETH"
+  },
+  "buyOther": {
+    "message": "Buy $1 or deposit from another account."
+  },
```

Comment:
This should include a description property that describes what the variable $1 is.

Reply:
only this fix...otherwise LGTM